tests/Unit/Services/VoucherEvaluator/VoucherEvaluatorTest.php

```diff
@@ -90,34 +154,65 @@ public function itCreditsWhenAFamilyIsPregnant()
     }
 
     /** @test */
-    public function itDoesNotCreditWhenAFamilyIsIneligible()
+    public function itCreditsUnrestrictedPrimarySchoolChildren()
     {
-        // Make extended evaluator
-        $evaluator = EvaluatorFactory::make("extended_age");
+        // get rules mods
+        $rulesMods = collect($this->rulesMods["credit-extended"]);
+
+        // Make evaluator
+        $evaluator = EvaluatorFactory::make($rulesMods);
 
         // Make our family
         $family = factory(Family::class)->create();
 
-        // Make a set of ineligible kids (no under school age)
-        $overSchool = factory(Child::class, 'overSchoolAge')->make();
-        $overSecondarySchool = factory(Child::class, 'overSecondarySchoolAge')->make();
+        // Make kids
+        $isPrimarySchool = factory(Child::class, 'isPrimarySchoolAge')->make();
+        $isOverPrimarySchool = factory(Child::class, 'isSecondarySchoolAge')->make();
 
         // Add the kids and check they saved
-        $family->children()->saveMany([$overSchool ,$overSecondarySchool]);
+        $family->children()->saveMany([$isPrimarySchool ,$isOverPrimarySchool]);
+        $this->assertEquals(2, $family->children()->count());
+
+        $evaluation = $evaluator->evaluate($family);
+
+        // Check it can find eligible children (0 vouchers)
```

Comment:
I think... The primary kids should get 3 here. And does not matter about the under school age. Alter comment?